noxfile.py

```diff
@@ -216,6 +216,7 @@ def lint(session):
     )
     session.run("mypy", ".", "--strict", silent=SILENT)
     session.run("flake8", "--config", ".flake8")
+    session.run("yamllint", ".")
```

Comment:
can you remove the installation of black from line 200?
I don't think it's needed.

Reply:
Will push once these changes pass on CI :)